devicetypes/smartthings/zigbee-window-shade-battery.src/zigbee-window-shade-battery.groovy

```diff
@@ -189,7 +189,12 @@ def setLevel(data, rate = null) {
 
 def pause() {
 	log.info "pause()"
-	zigbee.command(CLUSTER_WINDOW_COVERING, COMMAND_PAUSE)
+	// If the window shade isn't moving when we receive a pause() command then just echo back the current state for the mobile client.
+	if (device.currentValue("windowShade") != "opening" && device.currentValue("windowShade") != "closing") {
+		sendEvent(name: "windowShade", value: device.currentValue("windowShade"), isStateChange: true, display: false)
+	} else {
+		zigbee.command(CLUSTER_WINDOW_COVERING, COMMAND_PAUSE)
```

Comment:
I'd suggest always sending the command because it doesn't hurt anything and it's possible the shade is moving but our current state isn't accurate for some reason.

Reply:
That's actually not a bad idea!